### ISSUE TITLE
Current Status

### ISSUE THREAD

**Issue by leonardoaraujosantos** [zhangjunh/DR-GAN-by-pytorch#4]
Hi @junhaozhang  and @zhangjunh  after starting those are the current results I got (1000 epochs) and training on cfp dataset
## Loss Discriminator and Generator
![lossgeneratordiscriminator](https://user-images.githubusercontent.com/898383/42987234-6ab5fb50-8bf0-11e8-893f-8f2a32a3baf2.png)

## Some result
![epoch1340_253_10](https://user-images.githubusercontent.com/898383/42987240-6f92ee8a-8bf0-11e8-90f9-725c892c6b36.jpg)

## Questions
* Those results/loss are consistent for the epoch 1000 (I know that I should wait 10x more but my titanX at home takes 1000epochs/day)
* If everything is how it suppose to the I will add another PR for the tensorboardX
* I can also create a small script using dlib to centralize the faces and give it a try

Btw glad to help if you guys would like me to test/develop something please just shout

**Comment by zhangjunh** [comment]
Hi @leonardoaraujosantos , I noticed a bug which linked my old username (junhaozhang) to a stranger as a contributor, so I fixed it and you can just @zhangjunh in the future.
I don't know why the G and D can't converging together, same thing happened when I was training it. I ascribed this problem to the inappropriate hyper-parameters, such as learning-rates.
If you want, just add a PR with tensorboardX and the data-set which centralized the faces.
